### PR TITLE
migrate badge documentation from pull into wiki

### PR DESCRIPTION
**Pull request by mccrossen**
from this pull https://github.com/louislam/uptime-kuma/pull/1119

### CODE REVIEW

File: Badge.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping and uptime badges, generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are three API endpoints that return an SVG graphic for a given (public) monitor:
+
+**Status** `<kuma-url>/api/badge/:monitorID/status` e.g. `<kuma-url>/api/badge/1/status` returning sth like: ![image](img/badge/148052351-08a5713f-2bdf-4034-ab2a-29c7638cb25c.png)
```
Comment (CommanderStorm) [inline Badge.md:15]: I think this would be clearer if presented as a table (=> less visual clutter)

File: Badge.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+
+**Ping** `<kuma-url>/api/badge/:monitorID/ping/:duration` e.g. `<kuma-url>/api/badge/1/ping` returning sth like: ![image](img/badge/148052406-55b55a76-f61b-4a2a-a036-53b557eebc49.png)
+
+**Uptime** `<kuma-url>/api/badge/:monitorID/uptime/:duration` e.g. `<kuma-url>/api/badge/1/uptime/24` returning sth like: ![image](img/badge/147946965-5a706c7f-1efe-4056-b06b-b149904f87b5.png)
+
+# Status badge
```
Comment (CommanderStorm) [inline Badge.md:21]: I think we can skip these explainations and instead show the badge-maker. Currently, there are a few badges which are not included in this explaination like cert-expiry

Said feature is hidden here:
image
image
image

File: Badge.md [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+- [Status badge](#status-badge)
+- [Ping and uptime badge](#ping-and-uptime-badge)
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping and uptime badges, generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are three API endpoints that return an SVG graphic for a given (public) monitor:
```
Comment (CommanderStorm) [inline Badge.md:13]: Please include all badges, not only the ones from the intial PR
image

**Review by CommanderStorm — CHANGES_REQUESTED** [review]
Thanks for the PR, but unfortunately the information in said PR is a bit outdated in some places.
I have made some inline comments where presenting more/less information would be advantageous.

File: Badge.md [hunk 4]
```diff
@@ -1,3 +1,94 @@
...
+
+## Open Badge Generator
+
+Call up the Open Badge Generator of a service by pressing the Settings button on a status page in edit mode.
+
+![image](img/badge/settings-button.png)
+
+![image](img/badge/settings.png)
+
+![image](img/badge/open-badge-generator.png)
```
Comment (CommanderStorm) [inline Badge.md:88]: I think including this as a tip at the very top of the page would be better.
I have reworded some content to make this into a tip

Tips look like this:
image

```suggestion
> [!TIP]
> We have an Badge Generator integrated which makes this configuration more interactive.
> 
> You can use said generator via: 
> - navigating to a status page in edit mode
> - <details><summary>Clicking on the settings icon</summary>
>   <p>
>   
>   ![image](img/badge/settings-button.png)
>   
>   </p>
>   </details>
> - <details><summary>Clicking on the <code>Open Badge Maker</code>-button</summary>
>   <p>
>   
>   ![image](img/badge/settings.png)
>   
>   </p>
>   </details>
> - <details><summary>filling out the form to create your own badge</summary>
>   <p>
>   
>   ![image](img/badge/open-badge-generator.png)
>   
>   </p>
>   </details>
```

File: Badge.md [hunk 5]
```diff
@@ -1,3 +1,94 @@
...
+
+`<kuma-url>/api/badge/<monitorId>/status?style=flat-square`
+
+## Certificate Expire badge
+
+There are options to customize the **cert-exp** badge's text: `labelPrefix`, `label`, `labelSuffix`, `prefix`, `suffix = days`, `color`, `labelColor`, `upColor`, `downColor`, `warnDays`, `downDays`:
+
+## Response badge
+
+There are options to customize the **response** badge's text: `labelPrefix`, `label`, `labelSuffix`, `prefix`, `suffix = ms / %`, `color`, `labelColor`:
```
Comment (CommanderStorm) [inline Badge.md:74]: I don't have any good ideas how to make this feel less like a (quite confusing) parameter dump.

I think these could work:
- explaining all the common options only once (`labelPrefix`, `label`, `labelSuffix`, `prefix`, `color`, `labelColor`). `Badge Styles` from below should be handled like this as well.
- Using a table for the remaining options

File: Badge.md [hunk 6]
```diff
@@ -1,3 +1,94 @@
...
+  - [Badge Styles](#badge-styles)
+  - [Open Badge Generator](#open-badge-generator)
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping, uptime, avg-response badges, cert-exp and response generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are six API endpoints that return an SVG graphic for a given (public) monitor:
```
Comment (CommanderStorm) [inline Badge.md:17]: Nit: Lets avoid adding a number that we need to update here ^^
(if we add another badge in the future and don't update this number, people will think we are morons that can't count to 7...)

```suggestion
These are the endpoints that return an SVG graphic for a given (public) monitor:
```

**Review by CommanderStorm — CHANGES_REQUESTED** [review]
Thanks for including the other badges.
Looks much more mergeable now.

I have made a few discussions/suggestions, how we can hopefully make this even less confusing.

File: Badge.md [hunk 7]
```diff
@@ -1,3 +1,94 @@
...
+![image](img/badge/settings-button.png)
+
+![image](img/badge/settings.png)
+
+![image](img/badge/open-badge-generator.png)
+
+# Screenshots
+
+![image](img/badge/examples.png)
+
```
[the remark this answers is not in the document]
Comment (mccrossen) [reply Badge.md:95]: we can delete it, there are enough pictures in the documentation

File: Badge.md [hunk 8]
```diff
@@ -1,3 +1,94 @@
...
+|--------------|-------------------------------------------------------|-------------------------------------|----------------------------------------------|
+| status       | <kuma-url>/api/badge/:monitorID/status                | <kuma-url>/api/badge/1/status       | ![image](img/badge/status-example.png)       |
+| uptime       | <kuma-url>/api/badge/:monitorID/uptime:duration       | <kuma-url>/api/badge/1/uptime       | ![image](img/badge/uptime-example.png)       |
+| ping         | <kuma-url>/api/badge/:monitorID/ping:duration         | <kuma-url>/api/badge/1/ping         | ![image](img/badge/ping-example.png)         |
+| avg-response | <kuma-url>/api/badge/:monitorID/avg-response:duration | <kuma-url>/api/badge/1/avg-response | ![image](img/badge/avg-response-example.png) |
+| cert-exp     | <kuma-url>/api/badge/:monitorID/cert-exp              | <kuma-url>/api/badge/1/cert-exp     | ![image](img/badge/cert-exp-example.png)     |
+| response     | <kuma-url>/api/badge/:monitorID/response              | <kuma-url>/api/badge/1/response     | ![image](img/badge/response-example.png)     |
```
[the remark this answers is not in the document]
Comment (mccrossen) [reply Badge.md:28]: ok, but why not at `Status` and `Uptime` at `URL-Schema` and `URL-Example`?

[the remark this answers is not in the document]
Comment (CommanderStorm) [reply Badge.md:28]: What?

**Comment by mccrossen** [comment]
@CommanderStorm I have reorganised the entire content, I hope it is now easier to read and understand. What's your opinion?

**Summary**
- The open badge generator is very important, that's why I put it at the beginning
- The screenshot has been deleted
- The options are reorganised as a list

File: Badge.md [hunk 9]
```diff
@@ -1,3 +1,139 @@
...
-Please read: https://github.com/louislam/uptime-kuma/pull/1119
\ No newline at end of file
+- [Description](#description)
+- [Open Badge Generator](#open-badge-generator)
+- [Badge Type Options](#badge-type-options)
+  - [Status badge](#status-badge)
+  - [Ping, uptime, avg-response and response badge](#ping-uptime-avg-response-and-response-badge)
+    - [Badge Duration at ping, uptime and avg-response](#badge-duration-at-ping-uptime-and-avg-response)
+  - [Certificate Expire badge](#certificate-expire-badge)
+- [Badge Styles](#badge-styles)
```
Comment (CommanderStorm) [inline Badge.md:11]: Rationale: other pages of the wiki also don't have such a section

```suggestion
```

File: Badge.md [hunk 10]
```diff
@@ -1,3 +1,139 @@
...
+|----------------|---------------------------------------------------------|---------------------------------------|----------------------------------------------|
+| `status`       | `<kuma-url>/api/badge/:monitorID/status`                | `<kuma-url>/api/badge/1/status`       | ![image](img/badge/status-example.png)       |
+| `uptime`       | `<kuma-url>/api/badge/:monitorID/uptime:duration`       | `<kuma-url>/api/badge/1/uptime`       | ![image](img/badge/uptime-example.png)       |
+| `ping`         | `<kuma-url>/api/badge/:monitorID/ping:duration`         | `<kuma-url>/api/badge/1/ping`         | ![image](img/badge/ping-example.png)         |
+| `avg-response` | `<kuma-url>/api/badge/:monitorID/avg-response:duration` | `<kuma-url>/api/badge/1/avg-response` | ![image](img/badge/avg-response-example.png) |
+| `cert-exp`     | `<kuma-url>/api/badge/:monitorID/cert-exp`              | `<kuma-url>/api/badge/1/cert-exp`     | ![image](img/badge/cert-exp-example.png)     |
+| `response`     | `<kuma-url>/api/badge/:monitorID/response`              | `<kuma-url>/api/badge/1/response`     | ![image](img/badge/response-example.png)     |
+
+# Open Badge Generator
+
```
Comment (CommanderStorm) [inline Badge.md:32]: Don't think this tip needs a title ^^

```suggestion
```

File: Badge.md [hunk 11]
```diff
@@ -1,3 +1,139 @@
...
+- `upLabel`,Default value: Up
+- `downLabel`, Default value: Down
+- `upColor` 
+- `downColor`
+
+**Example**
+
+`<kuma-url>/api/badge/<monitorId>/status?upLabel=<upLabel>&downLabel=<downLabel>` ![image](img/badge/status-label.png)
+
+`<kuma-url>/api/badge/<monitorId>/status?upColor=white&downColor=black` ![image](img/badge/status-bw.png)
```
Comment (CommanderStorm) [inline Badge.md:74]:
```suggestion
**Examples**

| Badge | Url |
---------------
| ![image](img/badge/status-label.png) | `<kuma-url>/api/badge/<monitorId>/status?upLabel=<upLabel>&downLabel=<downLabel>` |
| ![image](img/badge/status-bw.png) | `<kuma-url>/api/badge/<monitorId>/status?upColor=white&downColor=black` |
```

File: Badge.md (hunk 9 above)
Comment (CommanderStorm) [inline Badge.md:11]: I think this can be pruned as the other wiki pages also don't have this

```suggestion
```

File: Badge.md [hunk 12]
```diff
@@ -1,3 +1,139 @@
 (Version >= 1.16.0)
+With https://github.com/louislam/uptime-kuma/pull/1119
```
Comment (CommanderStorm) [inline Badge.md:2]: Other sections of the wiki also don't have the PR they are added in

```suggestion
```

File: Badge.md [hunk 13]
```diff
@@ -1,3 +1,139 @@
...
+
+`<kuma-url>/api/badge/<monitorId>/ping/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>`
+
+![image](img/badge/ping-label.png)
+
+`<kuma-url>/api/badge/<monitorId>/uptime/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>`
+
+![image](img/badge/uptime-label.png)
+
+`<kuma-url>/api/badge/<monitorId>/avg-response/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>`
```
Comment (CommanderStorm) [inline Badge.md:97]: > [!NOTE]
> there is no example image for the third link

```suggestion
**Examples**

| Badge | Url |
---------------
| ![image](img/badge/ping-label.png) | `<kuma-url>/api/badge/<monitorId>/ping/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>` |
| ![image](img/badge/uptime-label.png) | `<kuma-url>/api/badge/<monitorId>/uptime/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>` |
|  | `<kuma-url>/api/badge/<monitorId>/avg-response/24?labelPrefix=<labelPrefix>&label=<label>&labelSuffix=<labelSuffix>&prefix=<prefix>&suffix=<suffix>` |
```

File: Badge.md [hunk 14]
```diff
@@ -1,3 +1,139 @@
...
+
+The default uptime badge's color is based on the uptime percentage:
+
+![image](img/badge/uptime-color.png)
+
+but can be overwritten with predefined color or any other color in HEX code:
+
+![image](img/badge/custom-colours.png)
+
+See https://www.npmjs.com/package/badge-maker#colors for predefined colors.
```
Comment (CommanderStorm) [inline Badge.md:107]:
```suggestion
but can be overwritten with [predefined color](https://www.npmjs.com/package/badge-maker#colors) or any other color in HEX code:

![image](img/badge/custom-colours.png)
```

File: Badge.md [hunk 15]
```diff
@@ -1,3 +1,139 @@
...
+
+### Badge Duration at ping, uptime and avg-response
+
+The given time interval at `Badge Duration (in hours)` has to be in 'hours', but customizing options allow for a different display: 
+
+`<kuma-url>/api/badge/1/uptime/720?label=30&labelSuffix=d`
+
+![image](img/badge/duration.png)
```
Comment (CommanderStorm) [inline Badge.md:115]:
```suggestion
**Example***

| Badge | Url |
---------------
| ![image](img/badge/duration.png) | `<kuma-url>/api/badge/1/uptime/720?label=30&labelSuffix=d` |
```

File: Badge.md [hunk 16]
```diff
@@ -1,3 +1,139 @@
...
+- `flat-square`
+- `plastic`
+- `for-the-badge`
+- `social`
+
+**Example**
+
+`<kuma-url>/api/badge/<monitorId>/status?style=flat-square`
+
+![image](img/badge/badge-style.png)
```
Comment (CommanderStorm) [inline Badge.md:139]:
```suggestion
**Example**

| Badge | Url |
---------------
| ![image](img/badge/badge-style.png) | `<kuma-url>/api/badge/<monitorId>/status?style=flat-square` |
```

**Review by CommanderStorm — APPROVED** [review]
I would be fine with the current version.
I have made a few comments below where I think this can be made to be more concise.

File: Badge.md (hunk 12 above)
Comment (mccrossen) [reply Badge.md:2]: ok, so we delete here

File: Badge.md (hunk 10 above)
Comment (mccrossen) [reply Badge.md:31]: ok, so we delete this headline

File: Badge.md (hunk 9 above)
Comment (mccrossen) [reply Badge.md:10]: you are right, a toc is no longer needed at github.com, there is the "Outline" feature, which creates the same thing.

[https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)

**Comment by CommanderStorm** [comment]
I don't see why you requested a review. My last review was approval

> I would be fine with the current version.

File: Badge.md [hunk 17]
```diff
@@ -1,3 +1,125 @@
...
+
+but can be overwritten with [predefined color](https://www.npmjs.com/package/badge-maker#colors) or any other color in HEX code:
+
+![image](img/badge/custom-colours.png)
+
+### Badge Duration at ping, uptime and avg-response
+
+The given time interval at `Badge Duration (in hours)` has to be in 'hours', but customizing options allow for a different display: 
+
+**Example***
```
Comment (CommanderStorm) [inline Badge.md:97]: Noticed there was a typo added by me ^^

```suggestion
**Example**
```

**Pull request merged**